### ISSUE TITLE
Standardize link color text

### ISSUE THREAD

**Issue by MattDodsonEnglish** [libremfg/rhize-docs#16]
Currently the content links are blue and sidebar links are green.
Pretty ugly looking:
![image](https://github.com/libremfg/libremfg.github.io/assets/47385188/2f7ffdee-c883-43e3-bc04-f73bf1f90b88)

 I think it's from a remote bootstrap CSS that overrides the content. If there are no design or accessibility issues, ideally all should be green.

**Comment by MattDodsonEnglish** [comment]
Fixed with #164
